.github/workflows/build-cli.yml

```diff
@@ -29,16 +29,27 @@ jobs:
       - name: Upload the Tekton definitions
         uses: actions/upload-artifact@v2
         with:
-          name: ibm-mas-tekton-${{ env.VERSION }}.yaml
-          path: ${{ github.workspace }}/image/cli/mascli/templates/ibm-mas-tekton.yaml
+          name: ibm-mas-tekton-${{ env.VERSION }}.yml
+          path: ${{ github.workspace }}/image/cli/mascli/templates/ibm-mas-tekton.yml
```

Comment:
Wouldn't change this, turns this into a breaking change.  we use `yml` inside ansible code, and `yaml` for things that a user will `oc apply -f` to align with what seems or seemed at the time, the generally accepted standard in both contexts.

Reply:
The two PRs from ansible-devops mentioned in the description of the PR must be merged first :+1:. I will emphasize they need to be merged. I will remove the zip, goo catch!

Reply:
Done. All extensions revisited following our chat in slack:
- Keep `yml.j2` for all `j2` files
- Everything else `yaml` except by:
- generate-tekton playbook, kept as `yml` (as it is ansible)